README.md

```diff
@@ -462,6 +462,22 @@ keys are delegated attributes' names and values are keys on synced data Hash. Th
 version of `delegate :name, to: :synced_data` which works with Hash reserved attributes names, like
 `:zip`, `:map`.
 
+## Persisting fetched objects
```

Comment:
Well, ok, we can mention that to be explicit, but I think no one really cares here and I don't see why would we even have auto pagination feature, so good catch, that can be removed some time later.